packages/mui-material/src/Divider/Divider.js

```diff
@@ -137,6 +137,8 @@ const DividerRoot = styled('div', {
         whiteSpace: 'nowrap',
         textAlign: 'center',
         border: 0,
+        borderTopStyle: 'solid',
```

Comment:
this change sets the default border style, that can be inherited by before and after pseudo-elements.

Reply:
What happens if we remove these? Isn't l[ine 61 above](https://github.com/mui/material-ui/pull/42715/files#diff-86967ad438e02627f9b54f7a5f0f9de83131e1a2077da8b3530054213e52664bR61) enough to set the default to be inherited?

Reply:
In our case (divider with children), the border-style is being inherited from Line 139, not from Line 61. Line 139, `border : 0` sets the border-style to `none`, which will remove the border. Hence, the need to explicitly add border styling.

Reply:
I see, this makes sense 👍🏼